docs/pbc/all/payment-service-provider/202410.0/base-shop/third-party-integrations/payone/app-composition-platform-integration/payment-method-flows/pay-pal-express.md

```diff
@@ -0,0 +1,184 @@
+---
+title: Payone PayPal Express in a headless application
```

Comment:
What is the purpose of this document? It describes the flow but it also provides some installation steps

Reply:
It's for describing the flow and which packages are required to implement this approach. Do you think it should be splitted and the installation part should be moved outside?

For me, it should be like this and we should probably update the other pages later the same way. The reasoning is that projects may have already been installed at some point and this kind of an addon.